### ISSUE TITLE
Cannot register buffer element provider when terminal has not been opened yet

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#173867]
In VS Code exploration devtools console after update.

```
ERR Cannot register buffer element provider when terminal has not been opened yet: Error: Cannot register buffer element provider when terminal has not been opened yet
    at P.registerBufferElementProvider (vscode-file://vscode-app/Applications/Visual%20Studio%20Code%20-%20Exploration.app/Contents/Resources/app/node_modules.asar/xterm/lib/xterm.js:1:24234)
    at e.Terminal.registerBufferElementProvider (vscode-file://vscode-app/Applications/Visual%20Studio%20Code%20-%20Exploration.app/Contents/Resources/app/node_modules.asar/xterm/lib/xterm.js:1:274582)
    at vscode-file://vscode-app/Applications/Visual%20Studio%20Code%20-%20Exploration.app/Contents/Resources/app/out/vs/workbench/workbench.desktop.main.js:2508:33396
```

**Comment by meganrogge** [comment]
Immediately after creating a terminal, run `terminal: focus accessible buffer`. Verify that you don't see this message in dev tools.
